ambient_toolbox/utils/model.py

```diff
@@ -29,3 +29,41 @@ def object_to_dict(obj, blacklisted_fields: list = None, include_id: bool = Fals
             valid_data[key] = value
 
     return valid_data
+
+
+def get_cached_related_obj(obj: Model, related_field_name: str, silently_return_none: bool = False):
```

Comment:
issue: could you add the relevant parts of the docstring to the docs, please? ❤️

Reply:
done

Reply:
❤️